js/user-form.js

```diff
@@ -0,0 +1,103 @@
+const HASHTAG_MAX_LENGTH = 20;
+const HASHTAG_MAX_COUNT = 5;
+const HASHTAG_REGULAR_EXPRESSION = /^#[A-Za-zА-Яа-яёЁ0-9]{1,20}$/;
+
+const imgUpLoadForm = document.querySelector('.img-upload__form');
+const upLoadPhoto = document.querySelector('#upload-file');
+const inputFile = document.querySelector('.img-upload__input');
+const imgUpLoadOverlay = document.querySelector('.img-upload__overlay');
+const imgUpLoadCancel = document.querySelector('#upload-cancel');
+
+const fieldHashtags = document.querySelector('.text__hashtags');
+const fieldDescription = document.querySelector('.text__description');
+
+const escapeKey = (evt) => evt.key === 'Escape';
+
+//отключение закрытия формы по ESC при фокусе на хэштеге или на комментарии
+const onFormEscKey = (evt) => {
+  if(escapeKey(evt) && document.activeElement !== fieldHashtags && document.activeElement !== fieldDescription) {
+    evt.preventDefault();
+    offLoadNewPhoto();
+  }
+};
+
+//Функция добавления обработчика закрытие формы по ESC
+const addUserFormEscapeKey = () => {
+  document.addEventListener('keydown', onFormEscKey);
+};
+
+//Функция удаления обработчика закрытие формы по ESC
+const removeUserFormEscapeKey = () => {
+  document.removeEventListener('keydown', onFormEscKey);
+};
+
+function onLoadNewPhoto () {
+  imgUpLoadOverlay.classList.remove('hidden');
+  document.body.classList.add('modal-open');
+  addUserFormEscapeKey();
+}
+
+function offLoadNewPhoto () {
+  imgUpLoadOverlay.classList.add('hidden');
+  document.body.classList.remove('modal-open');
+  removeUserFormEscapeKey();
+  inputFile.value = '';
+  fieldHashtags.value = '';
+  fieldDescription.value = '';
+}
+
+const pristine = new Pristine(imgUpLoadForm, {
+  classTo:  'img-upload__text-field',
+  errorTextParent: 'img-upload__text-field'});
```

Comment:
Попробуй так
```suggestion
const pristine = new Pristine(imgUpLoadForm, {
  classTo: 'img-upload__form',
  errorClass: 'img-upload__form--invalid',
  successClass: 'img-upload__form--valid',
  errorTextParent: 'img-upload__text',
  errorTextClass: 'img-upload__text-error',
  errorTextTag: 'div',
});
```

И можно цвет текста ошибки поменять в стилях